arcsi/api/show.py

```diff
@@ -84,6 +88,17 @@ def make_show(self, data, **kwargs):
 @arcsi.route("/show", methods=["GET"])
 @arcsi.route("/show/all", methods=["GET"])
 def list_shows():
+    do = DoArchive()
+    shows = Show.query.all()
+    for show in shows:
+        if show.cover_image_url:
+            show.cover_image_url = do.download(
+                show.archive_lahmastore_base_url, show.cover_image_url
+            )
+    return shows_schema.dumps(shows)
+
+@arcsi.route("/show/all_schedule", methods=["GET"])
```

Comment:
That's an interesting idea, but if we do this way then we will lose the response examples belonging to the different API calls in Swagger, what do you think about this?

Reply:
> We are basically slicing the same data, so most of the difference would be in schema applied. We could still control the flow with if-expressions. For example: /item/all?by=latest (diff. example: /item/all?by=2020-04-03) or /show/all?display=schedule (diff.: /show/all?display=list) etc. This is just a conversation starting idea let's not rush defining this well.

This would look more logical to me, I guess.